validation-test/compiler_scale/scale_neighbouring_getset.gyb

```diff
@@ -1,5 +1,5 @@
 // RUN: %scale-test --sum-multi --typecheck --begin 5 --end 16 --step 5 --select typeCheckAbstractFunctionBody %s
-// REQUIRES: OS=macosx, tools-release, assertions
+// REQUIRES: OS=macosx
```

Comment:
I'm pretty sure you need assertions on in order to get the statistics to be compiled-in. This will probably fail testing.